buildSrc/src/main/groovy/com/thoughtworks/go/build/WindowsPackagingTask.groovy

```diff
@@ -73,10 +74,18 @@ class WindowsPackagingTask extends DefaultTask {
       downloadAction.execute()
 
       project.copy {
-        from project.tarTree(project.fileTree("${winRootDir()}/jdk-download").include("*").singleFile)
+        from project.tarTree(project.fileTree("${winRootDir()}/jdk-download").singleFile)
         into buildRoot()
       }
 
+      FileTree tree = project.fileTree(buildRoot())
+      tree.each { File file ->
+        if (file.name.matches("src.zip")) {
+          println("Removing src.zip from JDK...")
+          project.delete(file)
+        }
+      }
```

Comment:
#### Agent Installer Size:

##### Prior to deletion of `src.zip`
```shell
➜  gocd git:(install-openjdk-on-windows) ✗ ls -alh installers/target/distributions/win
total 371808
drwxrwxr-x  3 ganeshpl  staff   102B Nov 14 14:47 .
drwxrwxr-x  3 ganeshpl  staff   102B Nov 14 14:46 ..
-rw-rw-r--  1 ganeshpl  staff   182M Nov 14 14:47 go-agent-18.11.0-7931-jre-64bit-setup.exe
```

##### After deletion of `src.zip`
```shell
➜  gocd git:(install-openjdk-on-windows) ls -alh installers/target/distributions/win
total 270272
drwxrwxr-x  3 ganeshpl  staff   102B Nov 14 15:22 .
drwxrwxr-x  3 ganeshpl  staff   102B Nov 14 15:22 ..
-rw-rw-r--  1 ganeshpl  staff   132M Nov 14 15:22 go-agent-18.11.0-7931-jre-64bit-setup.exe
```